homeassistant/components/motioneye/config_flow.py

```diff
@@ -81,6 +87,10 @@ def _get_form(
                 cast(Dict[str, Any], reauth_entry.data) if reauth_entry else {}
             )
 
+        if self._hassio_discovery:
```

Comment:
Should it be triggered on empty dict?
```suggestion
        if self._hassio_discovery is not None:
```

Reply:
Sure.